frontend/projects/upgrade/src/app/features/dashboard/home/components/experiment-design/aliases-table/aliases-table.component.scss

```diff
@@ -5,27 +5,98 @@
     display: flex;  
     justify-content: space-between;
 
-    .icon-bump-left {
-      margin-left: -5px;
+    .title {
+      font-size: 18px;
+      font-weight: bold;
+    }
+    
+    .toggle-button {
+      margin-top: -5px;
+      .icon-bump-left {
+        margin-left: -5px;
+      }
+      .toggle-text {
+        margin-left: 5px;
+      }
     }
   }
 
   .alias-table {
-    width: 100%;
     height: 475px;
     overflow: auto;
 
+    mat-header-row {
```

Comment:
these `mat-` custom elements all get classes applied with the same name, like `.mat-header-row` and `.mat-row`, etc.  I've always read that classes are the preferred way to style things, and to avoid using element selectors and/or ids when possible to avoid confusing css rule-precedence issues. If you slap a dot in from of these style rules to make them class selectors instead of element selectors, I bet it will still work the same. can we try that?

Reply:
FYI, I didn't initially add the `mat-` elements in the `scss` file. They were already all over the stepper-related `scss` files. I can rename them all to `.mat-` if that's a better practice.

Reply:
it's up to you if you want to change it or not, there is a lack of style guidelines so it's definitely inconsistent, there are many places where `mat-` is used and many where `.mat-` is used. i just wanted to let you know about it.